.env.exampleV1AndV2.holesky

```diff
@@ -96,6 +99,12 @@ EIGENDA_PROXY_EIGENDA_V2_BLOB_PARAMS_VERSION=0
 # sent, decreasing this value is a crude sort of optimization.
 EIGENDA_PROXY_EIGENDA_V2_MAX_BLOB_LENGTH=1MiB
 
+# Address of the EigenDA service manager contract
+EIGENDA_PROXY_EIGENDA_V2_SERVICE_MANAGER_ADDR=0xD4A7E1Bd8015057293f0D0A557088c286942e84b
+
+# Address of the BLS operator state retriever contract
+EIGENDA_PROXY_EIGENDA_V2_BLS_OPERATOR_STATE_RETRIEVER_ADDR=0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7
```

Comment:
Config management for contract interactions getting more and more complex :\
Out of scope for this PR but @pakim249CAL had suggested having a registry contract that would hold the addresses of all our contracts. This would be a much nicer UX for users, since everywhere they only ever have to give a single contract address, and then all the other contract addresses are fetched automatically for them.

Reply:
An alternative which I've mentioned in the past is that we add a `--chain-id` or `--network` (given that we have both preprod and testnet on holesky) flag and that this sets all the correct addresses (that would be hardcoded in the binary for holesky and mainnet)